exercises/knapsack/metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Given a set of items, each with a weight, w and a value, v, determine the number of each item to include in a collection so that the total weight is less than or equal to a given limit and the total value is as large as possible"
```

Comment:
if the corresponding sentence in the readme changes because of other comments I made, remember to change this too

Reply:
Done!